frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/common/datatables/DataTablesUtils.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright Siemens AG, 2018. Part of the SW360 Portal Project.
+ *
+ * SPDX-License-Identifier: EPL-1.0
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.sw360.portal.common.datatables;
+
+import com.google.common.collect.ImmutableList;
```

Comment:
~~I thought we try to avoid guava, and use Java8 features instead? Do you need something specific which is not provided in Java8?~~ moved code

Reply:
Yes its only moved code. I don't want to refactor the guava usage in this PR.